lib/web_ui/lib/src/ui/painting.dart

```diff
@@ -785,3 +785,17 @@ class ImageDescriptor {
     return _createBmp(_data!, width, height, _rowBytes ?? width, _format!);
   }
 }
+
+class FragmentShader extends Shader {
+  FragmentShader({
+    required ByteBuffer spirv,
+    Float32List? floatUniforms,
+    bool debugPrint = false,
+  }) {
+    throw UnsupportedError('FragmentShader is not supported on web');
```

Comment:
Quick question here (I hope it is not disruptive): does this mean that `FragmentShader` is _also_ not supported using CanvasKit?

If no, I would (maybe) include it in the error message because otherwise it would be slightly misleading (the way it is phrased right now).

Reply:
For this PR at least, both are unsupported. This followed the style of the other UnsupportedErrors in this file, but I'll update to be more specific.